src/game/server/gamecontroller.cpp

```diff
@@ -747,7 +747,14 @@ void IGameController::Tick()
 				break;
 			case IGS_END_MATCH:
 				// start next match
-				CycleMap();
+				if(m_MatchCount >= m_GameInfo.m_MatchNum-1)
+				{
+					CycleMap();
+					break;
```

Comment:
this breaks the game in case sv_maprotation is empty.

Reply:
Thanks for the nice catch, oy!
I removed the break, now it works as intended, with and without sv_maprotation being set.